README.md

```diff
@@ -55,3 +61,56 @@ If your proposal was closed because a core contributor determined that it was
 not worth pursuing and you feel that it was wrongly closed, then feel free
 to join the developer IRC group and have a more in-depth discussion with 
 other core developers about the feature.
+
+## How core developers evaluate proposals
+
+The following is a list of considerations that core developers use when deciding
+to accept, close, or leave a proposal open. It is intended to be useful for core
+developers when considering proposals and for proposal-makers in drafting their
+proposals.
+
+**1. Does the proposal comply with the rules?**
```

Comment:
It may be better to use `###` headings instead of bold text, as some Markdown viewers don't break lines unless you add a blank line below.

Reply:
Done. :)
I used ``####`` to make the titles closer to normal size.